README.md

```diff
@@ -121,6 +121,7 @@ There are 2 extra required attributes:
 There are also two optional attributes:
 
 - `source_header`: Specify a particular email header to use to identify the source of a particular notification and assign it to the appropriate provider. If unset, `From` will be used, but if your emails are not received directly from the provider but instead pass through a mailing list or alias, you might need to set this to a different value such as `X-Original-Sender` instead.
+- `limit_emails_without_header_from`: Restrict the emails used as "From", maybe a email from a mailing list or alias, in order to limit the emails that are processed.
```

Comment:
This is still a non-obvious name to me, but I won't hold up the review about it. :-) Might be good though to add some more explanatory context here, similar to what you have in the CHANGELOG.md above?